tests/test_file_processor.py

```diff
@@ -76,17 +76,17 @@ def test_on_each_with_symlink(tmp_path: Path, monkeypatch: pytest.MonkeyPatch) -
     gitignore = Path(".gitignore")
     gitignore.write_text("/hidden_dir")
     # Create a directory with a symlink to a file in the same directory
-    dir = Path("dir")
-    dir.mkdir()
-    file1 = dir / "file"
+    dir_dir = Path("dir")
```

Comment:
It reads so derpy 🤣. What about `dir_path`

Reply:
Maybe should change the dir folder name to something else?

Reply:
Well it's only the instance here isn't it?

> Maybe should change the dir folder name to something else?

Sure, that would also help put it in context. Maybe `src` or `project`?

Reply:
Went with pkg.